department-of-veterans-affairs/caseflow
# Welcome page backend/frontend connection

Connects #5818 

### Description
This PR connects the welcome page's frontend to real backend data. It also adds some of the base backend functionality (adding out of service, authenticating users, etc).

### Testing Plan
1. Go to the rails console and run `RoSchedulePeriod.create!(user: User.first, start_date: 90.days.ago, end_date: 1.day.ago, file_name: 'fake file name')`
1. Load `/hearings/schedule/build` and confirm the schedule period is displayed correctly
1. Confirm setting out of service displays the out of service screen for all hearing schedule pages
1. Confirm an unauthenticated user can't access any hearing schedule pages
